src/AppBundle/GitHooks/Action/CheckForBlacklistedWords.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppBundle\GitHooks\Action;
+
+use CaptainHook\App\Config;
+use CaptainHook\App\Console\IO;
+use SebastianFeldmann\Git\Repository;
+use Symfony\Component\Process\Process;
+
+final class CheckForBlacklistedWords extends Action
+{
+    protected const ERROR_MESSAGE = 'You have left blacklisted keyword in your code!';
+
+    public function execute(Config $config, IO $io, Repository $repository, Config\Action $action): void
+    {
+        if (!$this->isEnabled($action)) {
+            return;
+        }
+
+        $files = $this->getChangedFiles($action, $repository);
+        if (empty($files)) {
+            return;
+        }
+
+        $filesArguments = implode(' ', $files);
+
+        $blacklistedKeywords = $action->getOptions()->get('keyword_blacklist', 'var_dump');
+        $blacklistedArguments = implode('|', $blacklistedKeywords);
+
+        $command = 'grep -iwnHE \''.$blacklistedArguments.'\' '.$filesArguments;
+        $process = new Process($command);
+
+        $process->run();
+
+        if ($process->getOutput()) {
+            $io->writeError("<error>{$process->getOutput()}</error>");
+            $this->throwError($action, $io);
+        }
+    }
+
+    private function getChangedFiles(Config\Action $action, Repository $repository)
```

Comment:
Missing return typehint.

Reply:
Done in 4bfa069b7532fda478f9dcc12aa48b0e923286a4